CHANGELOG.md

```diff
@@ -1,3 +1,99 @@
+# 94.4.0
+
+This minor release introduces new APIs to ActionSheet, BottomSheet and FlexibleHeader. Additionally, it includes new theming extension for TextControls component.
+
+## New features
+
+TextControls component now has a theming extension.
+
+```objc
+MDCFilledTextField *filledTextField = [[MDCFilledTextField alloc] init];
+MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+
+[filledTextField applyThemeWithScheme:containerScheme];
+```
+
+```swift
+let filledTextField = MDCFilledTextField()
+let scheme = MDCContainerScheme()
+
+filledTextField.applyTheme(withScheme: scheme)
+```
+
+## API changes
+
+### ActionSheet
+
+#### MDCActionSheetController
+
+*new* property: `contentEdgeInsets` in `MDCActionSheetController`
+
+*new* property: `imageEdgeInsets` in `MDCActionSheetController`
+
+### BottomSheet
+
+#### MDCBottomSheetController
+
+*new* property: `dismissOnDraggingDownSheet` in `MDCBottomSheetController`
+
+#### MDCBottomSheetPresentationController
+
+*new* property: `dismissOnDraggingDownSheet` in `MDCBottomSheetPresentationController`
+
+#### MDCBottomSheetTransitionController
+
+*new* property: `dismissOnDraggingDownSheet` in `MDCBottomSheetTransitionController`
+
+### FlexibleHeader
+
+#### MDCFlexibleHeaderView(ShiftBehavior)
+
+*new* property: `shiftedOffscreen` in `MDCFlexibleHeaderView(ShiftBehavior)`
+
+## Component changes
+
+## Changes
+
+### ActionSheet
+
+* [Add `contentEdgeInsets` API. (#9008)](https://github.com/material-components/material-components-ios/commit/8fb17f7b0ae42be323967d53ddcb43a18f9f1e7e) (Robert Moore)
+* [Add `imageEdgeInsets` API. (#9024)](https://github.com/material-components/material-components-ios/commit/52dbc6c0b7eec2cdb7214b3b146450ab9fec2f0e) (Robert Moore)
```

Comment:
I'm questioning this API's default values at this point, but since I couldn't do it all at once it had to land in pieces. @jverkoey Since this defaults to something other than `UIEdgeInsetsZero`, and that's sort of unexpected, do you think it should be reverted for now?

Reply:
I spoke offline with @randallli and we agreed that this PR should be rolled-back for this release. We can then discuss offline whether the APIs should default to `.zero` or if having non-`.zero` defaults is acceptable.